web/packages/core/src/ruffle-embed.ts

```diff
@@ -50,9 +50,11 @@ export class RuffleEmbed extends RufflePlayer {
                     allowScriptAccess,
                     src.value
                 ),
-                parameters: this.attributes.getNamedItem("flashvars")?.value,
-                backgroundColor: this.attributes.getNamedItem("bgcolor")?.value,
-                base: this.attributes.getNamedItem("base")?.value,
+                parameters:
+                    this.attributes.getNamedItem("flashvars")?.value ?? null,
```

Comment:
why is this needed? Isn't `Attr.value` typed as just a `string`?

Reply:
Yeah, but the optional chaning operator (`?.`) causes it to be `string | undefined`. `undefined` can happen if `getNamedItem("flashvars")` returns `null`.